src/data/pools/poolAPR.ts

```diff
@@ -0,0 +1,21 @@
+import { NetworkInfo } from 'constants/networks'
+
+export async function fetchPoolsAPR(networkInfo: NetworkInfo): Promise<{ [address: string]: number | undefined }> {
+  const result: { [address: string]: number | undefined } = {}
+  for (let i = 1; i < 100; i++) {
+    try {
+      const callResult = await fetch(
+        `${process.env.REACT_APP_POOL_SERVICE}/${networkInfo.poolRoute}/api/v1/elastic/pools?page=${i}&perPage=1000`
```

Comment:
put it into env.ts

Reply:
We dont have it yet bro. This is not dmm-interface